compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/HttpFiltersDesugar.java

```diff
@@ -155,18 +132,12 @@ private HttpFiltersDesugar(CompilerContext context) {
     }
 
     /**
-     * Check if the resource is an http resource and apply filter.
+     * Check if the resource is an http resource.
      *
      * @param resourceNode The resource to apply filter on if it is http
-     * @param env          the symbol environment
+     * @return true if the resource is an http resource
      */
-    void addHttpFilterStatementsToResource(BLangFunction resourceNode, SymbolEnv env) {
-        if (isHttpResource(resourceNode)) {
-            addFilterStatements(resourceNode, env);
-        }
-    }
-
-    private boolean isHttpResource(BLangFunction resourceNode) {
+    static boolean isHttpResource(BLangFunction resourceNode) {
```

Comment:
Shouldn't the docs for `resourceNode` be updated? Filters are not relevant for this check, right?